test/construct/integ.just-in-time-registration.test.ts

```diff
@@ -9,80 +9,113 @@ import {
-        code: new InlineCode('exports.handler = () => { return true; }'),
-        runtime: Runtime.NODEJS_12_X,
-        handler: 'index.js',
-      }),
-    ],
-    vault: {
-      bucket: bucket,
-      prefix: 'test',
-    },
+describe('Test index.ts importation', () => {
+  test('Create Functions', () => {
+    process.env.BASE_PATH = __dirname;
+    process.env.APPS_PATH = path.resolve(__dirname, '..', '..', 'src', 'lambda-assets');
+    const app = new App();
+    const stack = new Stack(app, 'test-stack');
+    const anotherStack = new Stack(app, 'another-stack');
+    const bucket = new Bucket(anotherStack, 'userProvidedBucket');
+    new DeviceActivator(stack, 'testDeviceActivator');
+    new VerifiersFetcher(stack, 'testVerifiersFetcher');
+    new CaRegistrator(stack, 'testCaRegistrationFunction', {
+      vault: {
+        bucket: bucket,
+        prefix: 'test',
+      },
+    });
+    new DeviceCertificateGenerator(stack, 'testDeviceCertificateGenerator', {
+      vault: {
+        bucket: bucket,
+      },
+    });
+    const reviewReceptor = new ReviewReceptor(stack, 'testReviewReceptor');
+    new ReviewAcceptionRole(reviewReceptor, 'testReviewAcceptionRole', 'iot.amazonaws.com');
+    new JitrTopicRule(reviewReceptor, 'testJitrTopicRule');
+    new RegistrationConfigRole(stack, 'testJitpRole');
   });
-
-  expect(SynthUtils.synthesize(stack).template).toMatchSnapshot();
-  expect(stack).toCountResources('AWS::Lambda::Function', 3);
-  expect(stack).toCountResources('AWS::IAM::Role', 4);
-  expect(stack).toHaveResourceLike('AWS::IAM::Role', {
-    RoleName: 'ReviewAcceptionRoleName-' + name,
-  });
-  expect(stack).toCountResources('AWS::SQS::Queue', 1);
 });
 
-test('CaRegisterApi integration test without providing verifiers and prefix', () => {
-  process.env.BASE_PATH = __dirname;
-  process.env.APPS_PATH = path.resolve(__dirname, '..', '..', 'src', 'lambda-assets');
-  const app = new App();
-  const stack = new Stack(app, 'test-stack');
-  const name = 'test-case';
-  const anotherStack = new Stack(app, 'another-stack');
-  const bucket = new Bucket(anotherStack, 'userProvidedBucket');
-  new JustInTimeRegistration(stack, name, {
-    vault: {
-      bucket: bucket,
-    },
+describe('Test JustInTimeRegistration', () => {
+  test('integration test', () => {
+    process.env.BASE_PATH = __dirname;
+    process.env.APPS_PATH = path.resolve(__dirname, '..', '..', 'src', 'lambda-assets');
+    const app = new App();
+    const stack = new Stack(app, 'test-stack');
+    const verifierStack = new Stack(app, 'verifier-stack');
+    const name = 'test-case';
+    const anotherStack = new Stack(app, 'another-stack');
+    const bucket = new Bucket(anotherStack, 'userProvidedBucket');
+    new JustInTimeRegistration(stack, name, {
+      verifiers: [
+        new Function(verifierStack, name, {
+          code: new InlineCode('exports.handler = () => { return true; }'),
+          runtime: Runtime.NODEJS_12_X,
```

Comment:
NODEJS_14_X or 16 support?

Reply:
1. There is no version 16 in CDK
2. FAIL  test/construct/integ.just-in-time-registration.test.ts (16.159 s)
  ● Test JustInTimeRegistration › integration test

    Inline source not allowed for nodejs14.x